nncf/version.py

```diff
@@ -9,8 +9,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-__version__ = "2.7.0"
+__version__ = "2.8.0"
```

Comment:
Should we bump the NNCF version to `2.9.0` on develop branch, @alexsu52, @MaximProshin?

Reply:
Yeah let's do it!

Reply:
Done! Review, please!